src/command/init/mod.rs

```diff
@@ -16,6 +16,8 @@ pub mod template_operations;
 #[cfg(feature = "composition-js")]
 pub mod transitions;
 
+#[cfg(feature = "composition-js")]
+use crate::command::init::options::ProjectType;
 #[cfg(feature = "composition-js")]
 use crate::command::init::options::{
     GraphIdOpt, ProjectNameOpt, ProjectOrganizationOpt, ProjectTypeOpt, ProjectUseCaseOpt,
```

Comment:
nit: can we include this with the other imports from `options`?
```suggestion
#[cfg(feature = "composition-js")]
use crate::command::init::options::{
    GraphIdOpt, ProjectNameOpt, ProjectOrganizationOpt, ProjectType, ProjectTypeOpt, ProjectUseCaseOpt,
```